minio/umbrel-app.yml

```
@@ -0,0 +1,34 @@
+manifestVersion: 1.1
```

This can be changed to version `1` since we don't have any hooks.

```suggestion
manifestVersion: 1
```